application/helpers/previewer_helper.php

```diff
@@ -228,7 +228,10 @@ function format_authors($authors, $flags = 0, $max = 0)
 
 function format_author($author, $flags = 0)
 {
-	return format_authors(array($author), $flags);
+	if (isset($author))
```

Comment:
This should work, but I think we need something on L148 as well.

Reply:
Currently the code at L148 (in function `format_author_name()`) is only ever called from `format_authors()`. When that function is called directly I would expect the array to just be empty if there are no authors such that `format_author_name()` will not get called. Although I can't say I am absolutely 100% confident about that...

The definite case we are seeing an issue with now is when null is passed into `format_author()` which then passes an array with one null element to `format_authors()` for which the added `isset` in `format_author()` is sufficient.

Reply:
Ah, right you are, apologies for the rushed review.